### PR TITLE
Core: New global constant: CLI

### PR DESCRIPTION
**Pull request by FrankFFFF**
Hi guys,
let me know if i should not create a global constant like this,
then i will change it to `PHP_SAPI === 'cli'` 
As my previous commit, i believe this is faster.

### CODE REVIEW

File: src/Core/functions.php [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 }
 
+if (!defined('CLI')) {
+    define('CLI', PHP_SAPI === 'cli');
```
Comment (markstory) [inline src/Core/functions.php]: We used to have a constant like this and we removed it as there wasn't much point given that `PHP_SAPI` and `php_sapi_name()` exist.

**Comment by ADmad** [comment]
No more unnecessary constants please.

File: src/Core/functions.php (hunk 1 above)
Comment (Oxicode) [inline src/Core/functions.php]: +1

**Comment by dereuromark** [comment]
If you want you can always have this constant in your user land code. But the core itself most likely does not need it. Closing then.

**Comment by Fieah** [comment]
Okay, will do `PHP_SAPI === 'cli'` later :)

**Pull request closed without merging**